libraries/Wire/examples/i2c_scanner/i2c_scanner.ino

```diff
@@ -68,10 +68,10 @@ void loop()
     byte error, address;
     int nDevices;
 
-    // Serial.println("Scanning...");
+    Serial.println("Scanning...");
 
     nDevices = 0;
-    for(address = 1; address < 127; address++) {
+    for(address = 1; address < 255; address++) {
```

Comment:
Did you test this with a device that truly has an 8-bit I2C address instead of the standard 7-bit address?

Reply:
Well, it's a mistakes made in test. Just ignore it.

Reply:
```suggestion
    for(address = 1; address < 127; address++) {
```